projects/sandbox/deployment/production/ingress.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
This will have to change as before, but we can handle that when we plan the migration for the production environment